SPECS/samba.spec

```diff
@@ -11,6 +6,11 @@
 # ctdb is enabled by default, you can disable it with: --without clustering
 %bcond_without clustering
 
+# XCP-ng build condition
+%bcond_without xcpng
```

Comment:
Didn't we say last Friday that there would be a commit to introduce this condition and restore parts removed by XenServer in their spec file, before the merge commit?